ch2driver/pytpcc/tpcc.py

```diff
@@ -333,24 +341,34 @@ def executorFunc(clientId, TAFlag, driverClass, qDone, warmupDurationQ, warmupDu
 
     if args['run_date']:
         run_date = args['run_date']
-        os.environ["RUN_DATE"] = str(run_date)
+    os.environ["RUN_DATE"] = str(run_date)
 
     if args['tls']:
         use_tls = '1'
     os.environ["TLS"] = use_tls
 
-    if args['unoptimized_queries']:
-        unoptimized_queries = '1'
-    os.environ["UNOPTIMIZED_QUERIES"] = unoptimized_queries
-
     if args["ignore_skip_index_hints"]:
         ignore_skip_index_hints = "1"
     os.environ["IGNORE_SKIP_INDEX_HINTS"] = ignore_skip_index_hints
 
+    schema = constants.CH2_DRIVER_SCHEMA["CH2"]
+    analyticalQueries = constants.CH2_DRIVER_ANALYTICAL_QUERIES["HAND_OPTIMIZED_QUERIES"]
```

Comment:
Are `schema`  and `analyticalQueries` being passed in when instantiating the driver for execution? For loading it looks like `schema` is, but not for execution.

Reply:
Fixed. Thanks for this comment. After some more thinking, made the fix more general to pass in schema and analyticalQueries to ONLY the driver. The loader and executor are agnostic to these parameters. The driver with which the loader and executor are created does the appropriate thing.